Inventory.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using InventoryManagementSystem.ProductInfo;
+
+namespace InventoryManagementSystem.InventoryInfo
+{
+    public class Inventory
+    {
+        private readonly int _Id;
+        public static int InventoryIdCounter { get; private set; }
+        public string InventoryName { get; set; }
+        private List<Product> ProductList;
+
+        public Inventory() : this("DefaultInventory")
+        {
+
+        }
+
+        public Inventory(string inventoryName)
+        {
+            InventoryName = inventoryName;
+            ProductList = new List<Product>();
+            _Id = InventoryIdCounter++;
+        }
+
+        public override string ToString() => $"Id {_Id} : Name = {InventoryName}";
+
+        public void AddProduct()
+        {
+            Console.Write("Product name : ");
+            var prodcutName = Console.ReadLine();
+            Console.Write("Product price : ");
+            var prodcutPrice = Console.ReadLine();
+            Console.Write("Product quantity : ");
+            var prodcutQuantity = Console.ReadLine();
+
+            var product = new Product()
+            {
+                ProductName = prodcutName,
+                ProductPrice = Double.Parse(prodcutPrice),
+                ProductQuantity = int.Parse(prodcutQuantity)
+            };
+            ProductList.Add(product);
+        }
+
+        public void ViewProdcuts()
+        {
+            foreach(var product in ProductList)
```

Comment:
NIT: `foreach (` . Please fix the format for all files, find that nice shortcut in your IDE that does that for you ;)